### PR TITLE
New Q2 messages for CloudBees

### PR DESCRIPTION
**Pull request by daniel-beck**
Links to the tester:

* [Jenkins World 2016](http://jenkinsci.github.io/patron/tester.html?caption=Jenkins+World+2016&link=https%3A%2F%2Fwww.cloudbees.com%2Fjenkinsworld%2Fhome&blurb=Save+the+Date%3A+September+13-15.+Jenkins+World+2016+is+the+event+for+everything+Jenkins%3A+community%2C+CloudBees+and+ecosystem.&logo=https%3A%2F%2Fpages.cloudbees.com%2Frs%2F083-PKZ-512%2Fimages%2FJenkins-World-Logo.png)
* [Become Jenkins Certified](http://jenkinsci.github.io/patron/tester.html?caption=Become+Jenkins+Certified&link=https%3A%2F%2Fwww.cloudbees.com%2Fjenkins-certification&blurb=The+first+certification+program+for+Jenkins+Engineers.&logo=https%3A%2F%2Fpages.cloudbees.com%2Frs%2F083-PKZ-512%2Fimages%2Fci_cert1.png)

Not to be merged before…
* Approved in project meeting
* image URLs are moved over to jenkins.io

### CODE REVIEW

File: messages.xml [hunk 1]
```diff
@@ -303,20 +303,22 @@
...
       <message>
-        <caption>Templating Jenkins Environments with Docker</caption>
-        <link>https://www.cloudbees.com/blog/templating-jenkins-build-environments-docker-containers</link>
-        <blurb>Read this Blog to Learn How</blurb>
-        <logo>https://jenkins.io/images/patrons/cloudbees.png</logo>
+        <caption>Become Jenkins Certified</caption>
+        <link>https://www.cloudbees.com/jenkins-certification</link>
+        <blurb>The first certification program for Jenkins Engineers.</blurb>
+        <logo>https://pages.cloudbees.com/rs/083-PKZ-512/images/ci_cert1.png</logo>
+        <!-- TODO to be copied over to jenkins.io for the duration of this message before publishing -->
```
Comment (rtyler) [inline messages.xml]: I don't understand this TODO?

Comment (daniel-beck) [inline messages.xml]: @rtyler I didn't think we want to allow sponsors to track visitors?

Comment (rtyler) [inline messages.xml]: Right, that I'm definitely in favor of doing. We should update them all at once IMO

Comment (daniel-beck) [inline messages.xml]: Past messages are never shown, and they've been hosted on jenkins.io/jenkins-ci.org for a while anyway: https://github.com/jenkins-infra/jenkins.io/commit/85310250220c20581a2ac343239729bba35218a6

**Comment by daniel-beck** [comment]
Approved in meeting of April 27.

@rtyler is this working?

**Comment by rtyler** [comment]
[INFRA-674](https://issues.jenkins-ci.org/browse/INFRA-674) is not yet resolved

**Pull request merged**